spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/RabbitServiceConfiguration.java

```diff
@@ -34,11 +35,12 @@
  * @author Mark Fisher
  * @author Dave Syer
  * @author Glenn Renfro
+ * @author David Turanski
  */
 @Configuration
 @ConditionalOnClass(RabbitMessageChannelBinder.class)
 @ConditionalOnMissingBean(RabbitMessageChannelBinder.class)
-@ImportResource("classpath*:/META-INF/spring-cloud-stream/binder/rabbit-binder.xml")
+@Import(RabbitMessageChannelBinderConfiguration.class)
 @PropertySource("classpath:/META-INF/spring-cloud-stream/rabbit-binder.properties")
```

Comment:
With `spring-boot` specific configuration properties, is this the right time to move all these default *-binder.properties into a centralized boot aware yaml file (src/main/resources/application.yml) ?